### PR TITLE
Showback measure migration 

### PR DESCRIPTION
**Pull request by aljesusg**
Migrate to measure_type

### CODE REVIEW

File: app/models/showback_measure_type.rb [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-  serialize :types, Array # Implement types column as an array
-  default_value_for(:types) { [] }
-
-  VALID_MEASURE_TYPES = %w(Occurence Throughput Frequency Capacity Integer).freeze
+  VALID_MEASURE_TYPES = %w(CPU).freeze
 
   validates :measure,
             :inclusion => { :in => VALID_MEASURE_TYPES }
 
+  VALID_TYPES = %w(average number).freeze
```
Comment (sergio-ocon) [inline app/models/showback_measure_type.rb]: ¿average_CPU, max_number_of_CPU?

Comment (aljesusg) [inline app/models/showback_measure_type.rb]: We need number to set de number of cores used of total of VM or Host

File: app/models/showback_measure_type.rb [hunk 2]
```diff
@@ -1,18 +1,19 @@
-class ShowbackConfiguration < ApplicationRecord
-  validates :name, :description, :name, :types, :presence => true
+class ShowbackMeasureType < ApplicationRecord
+  validates :name, :description, :measure, :types, :presence => true
 
-  serialize :types, Array # Implement types column as an array
-  default_value_for(:types) { [] }
-
-  VALID_MEASURE_TYPES = %w(Occurence Throughput Frequency Capacity Integer).freeze
+  VALID_MEASURE_TYPES = %w(CPU).freeze
```
Comment (sergio-ocon) [inline app/models/showback_measure_type.rb]: Measure Type: CPU:period_average
Category: CPU
Subcategory: period_average

¿Category?

Comment (sergio-ocon) [inline app/models/showback_measure_type.rb]: Or category: CPU
dimension: period_average

Then: we can store CPU:period_average or CPU + period_average

Comment (aljesusg) [inline app/models/showback_measure_type.rb]: See fixtures I set name  =  VM::CPU I don't understand . Why do you want categories?

Comment (sergio-ocon) [inline app/models/showback_measure_type.rb]: Because you are using Measure, that VM is not a measure ;)

**Pull request merged**